### DISCUSSION TITLE
How to update events with data from global.context

### DISCUSSION THREAD

**Discussion by diedder** [jensrossbach/node-red-contrib-chronos#76] [Q&A] [answered]
I'm using events with data stored in the global context. after deploying or restarting all nodes the schedule node imports the data from context, but when the context is updated / changed the node is not getting the new data. Is there an option to trigger a new import?

**Comment by jensrossbach** [comment, marked as answer]
Yes, you can send a message with a control command. Either set payload to string "reload" (this reloads data of all events or of the event in case of only one). Or set payload to an array and put the string "reload" to the index of the event you want to reload, all others can be set to null. This is also described in the wiki.

**Reply by diedder** [reply]
Thank you, I missed that in the wiki
